src/sync/Synchroniser.js

```diff
@@ -233,6 +240,7 @@ export class Synchroniser {
         method: 'POST',
         headers: {
           Authorization: this.authHeader,
+          msupply_site_uuid: uniqueId,
```

Comment:
The closest thing to a convention for HTTP header keys would be to use hyphens rather than under scores 
```suggestion
          msupply-site-uuid: uniqueId,
```

Reply:
Done.